carrot_goal_ratio: 1.0
max_ttc: 6.0
solver_time: 20
obey_time: true
allow_reverse: true
safety_radius: 0.2
adaptive_lookahead: false
odom_topic: "/odometry/controller_1"
cmd_vel_topic: "/cmd_vel"